scripts/enable_automatic_updates.sh

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env bash
+
+apt-get install unattended-upgrades curl
+
+echo '
+APT::Periodic::Update-Package-Lists "1";
+APT::Periodic::Unattended-Upgrade "1";
+' >/etc/apt/apt.conf.d/20auto-upgrades
+
+sed -i.bak 's|//Unattended-Upgrade::Remove-Unused-Dependencies "false"|Unattended-Upgrade::Remove-Unused-Dependencies "true"|' /etc/apt/apt.conf.d/50unattended-upgrades
+
+sed -i.bak 's|//Unattended-Upgrade::Automatic-Reboot "false"|Unattended-Upgrade::Automatic-Reboot "false"|' /etc/apt/apt.conf.d/50unattended-upgrades
+
+rm /etc/apt/apt.conf.d/50unattended-upgrades.bak
```

Comment:
If you're running this script (as opposed to manually working through it line by line) is there any point in using `.bak` in the `sed` command and then immediately removing the .bak file?

Reply:
True - .bak no longer needed!